tests/finance/test_slippage.py

```diff
@@ -1146,3 +1147,85 @@ def test_orders_stop(self, name, order_data, event_data, expected):
 
                 for key, value in expected['transaction'].items():
                     self.assertEquals(value, txn[key])
+
+
+class FixedBasisPointsSlippageTestCase(WithCreateBarData,
+                                       WithSimParams,
+                                       ZiplineTestCase):
+
+    START_DATE = pd.Timestamp('2006-01-05 14:31', tz='utc')
+    END_DATE = pd.Timestamp('2006-01-05 14:36', tz='utc')
+    SIM_PARAMS_CAPITAL_BASE = 1.0e5
+    SIM_PARAMS_DATA_FREQUENCY = 'minute'
+    SIM_PARAMS_EMISSION_RATE = 'daily'
+
+    ASSET_FINDER_EQUITY_SIDS = (133,)
+    ASSET_FINDER_EQUITY_START_DATE = pd.Timestamp('2006-01-05', tz='utc')
+    ASSET_FINDER_EQUITY_END_DATE = pd.Timestamp('2006-01-07', tz='utc')
+    minutes = pd.DatetimeIndex(
+        start=START_DATE,
+        end=END_DATE - pd.Timedelta('1 minute'),
+        freq='1min'
+    )
+
+    @classproperty
+    def CREATE_BARDATA_DATA_FREQUENCY(cls):
+        return cls.sim_params.data_frequency
+
+    @classmethod
+    def make_equity_minute_bar_data(cls):
+        yield 133, pd.DataFrame(
+            {
+                'open': [3.00],
+                'high': [3.15],
+                'low': [2.85],
+                'close': [3.00],
+                'volume': [200],
+            },
+            index=[cls.minutes[0]],
+        )
+
+    @classmethod
+    def init_class_fixtures(cls):
+        super(FixedBasisPointsSlippageTestCase, cls).init_class_fixtures()
+        cls.ASSET133 = cls.env.asset_finder.retrieve_asset(133)
+
+    def test_fixed_bps_slippage(self):
+
+        slippage_model = FixedBasisPointsSlippage()
```

Comment:
We should test that this actually respects the basis points value that it's given.  One way to do that would be to use a parameterized test, either with `parameter_space` or with `nose.parameterized`.

Reply:
This would also be a reasonable way to test different amounts.

Reply:
In that vein, probably worth having a test for an amount less than the volume limit.

Reply:
added these testes